modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -295,22 +295,25 @@ properties:
-            
+              Maximum depth to which a document's revision tree can grow, it governs the point at which to prune a document's revision tree. Revision pruning is the process to remove obsolete parent revisions and runs every time a revision is added.
+
+              The minimum value is 20.
+
+              ##### Consideration with Conflicting Revisions
+
+              Setting `revs_limit` to a low value (typically below 100) can have adverse effects with conflicting revisions.
+
+              If there are conflicting revisions, the document may end up with **disconnected branches** after the pruning process. On Sync Gateway, the pruning algorithm is applied to the shortest, non-tombstoned branch in the revision tree.
+
+              On the example below, the document has a conflicting branch (revisions `4'` - `1001'`). When the shortest branch reaches the 1003rd update, it gets cut off. The revision tree is not in a corrupted state and the logic that chooses the winning revision still applies. But it may make it impossible to do certain merges (n-way merge) to resolve conflicts and will occupy disk space that could have been free-ed if the conflict was resolved early on.<br><br>
+
               ![](https://cl.ly/3C1G3t3R1v19/pruning-sg.gif)
-              
-              If the revision tree gets into this state then the only option to resolve the conflict is to pick a winning branch and tombstone all the non-winning conflicting branches. It should be noted that tuning the `revs_limit` to a lower value increases the likelyhood of this scenario happening.
-              
+
+              If the revision tree gets into this state then the only option to resolve the conflict is to pick a winning branch and tombstone all the non-winning conflicting branches. Tuning the `revs_limit` to a lower value increases the likelyhood of this scenario happening.
+
               See also:
-              - Couchbase Lite [pruning](../../couchbase-lite/native-api/revision/index.html#pruning).
-              - Sync Gateway purge endpoint [/{db}/_purge](references/sync-gateway/admin-rest-api/index.html#!/document/post_db_purge).
-              - Sync Gateway [document TTLs](references/sync-gateway/admin-rest-api/index.html#!/document/put_db_doc).
+              - Sync Gateway purge endpoint [/{db}/_purge](admin-rest-api.html#/document/post__db___purge).
+              - Sync Gateway [document TTLs](admin-rest-api.html#/document/put__db___doc_).
             default: 1000
```

Comment:
Please update this based on earlier comment on default and min values.

Reply:
Also include reference to https://docs.couchbase.com/sync-gateway/2.6/config-properties.html#databases-foo_db-allow_conflicts